src/main/java/io/managed/services/test/Environment.java

```diff
@@ -119,6 +120,7 @@ public class Environment {
     public static final String PRIMARY_PASSWORD = getOrDefault(PRIMARY_PASSWORD_ENV, null);
 
     public static final String SECONDARY_USERNAME = getOrDefault(SECONDARY_USERNAME_ENV, null);
+    public static final String SECONDARY_PASSWORD = getOrDefault(SECONDARY_PASSWORD_ENV, null);
```

Comment:
remove this variable please. 
**why**:
this will fail: we do not provide Passwords when testing other than integration tests (which is still To remove once one issue in ocm will be resolved), PRIMARY_USERNAME & PRIMARY_PASSWORD are provided just because other "bug" which prevents us from logging into the cli with usage of offline tokens, therefore we provide this so user can still use this hack to log in to cli with username and passowrd.

Reply:
Fixed in: 8cd74dc